tempesta_fw/http.c

```diff
@@ -748,9 +748,15 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 		return r;
 
 	r = tfw_http_add_hdr_via(hm);
+	if (resp->cache_ctl.flags & TFW_HTTP_RESP_STALE) {
+		char *s_110_n = "Warning:";
+		char *s_110_v = "110 - Response is stale.";
```

Comment:
Perhaps, `"\"110 - Response is stale\""`? The text offered in the RFC does not contain the dot at the end, and does contain apostrophes.